src/main/java/org/jitsi/videobridge/EndpointConnectionStatus.java

```diff
@@ -54,48 +57,12 @@
     private final static String CFG_PNAME_BASE
         = "org.jitsi.videobridge.EndpointConnectionStatus";
```

Comment:
Is `CFG_PNAME_BASE` still used after this change?

Reply:
nope, removed